### ISSUE TITLE
Script Console: Link to Python section of the Help

### ISSUE THREAD

**Issue by probonopd** [aoloe/scribus-ui-concept#9]
It would be great if Script Console would have a menu entry or button that would open the Python section of the Scribus Help (jump directly to "Scripter API"), so that the user can go there directly from the Script Console rather than from the Help menu in the main Scribus window.

**Comment by aoloe** [comment]
yes, if one day script should get a good script editor (as written in the other ticket, personally, i'm not sure that it is a good idea), this help button should be its toolbar.
